test/spec/Feature/Query/InsertSpec.hs

```diff
@@ -477,7 +477,7 @@ spec actualPgVersion = do
             {"id": 204, "body": "yyy"},
             {"id": 205, "body": "zzz"}]|]
           `shouldRespondWith`
-          [json|{} |]
+          [json|{"code":"PGRST205","details":null,"hint":null,"message":"Could not find relation 'test.garlic' in the schema cache"} |]
```

Comment:
Hm, so the empty body `{}` on an insert error (https://github.com/PostgREST/postgrest/issues/3697) was tested. The behavior is a bit weird, it must have escaped review before.

Reply:
A quick `git blame` reveals that it was introduced in #2542.